From 5445d3424bd41eb6c1296a48104095777d44204d Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 28 Feb 2017 16:12:44 +0100
Subject: [PATCH] bridge: vlan: fix a few "fdb" typos in vlan doc

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1417289
Upstream Status: iproute2.git commit 296cee6fdfaab

commit 296cee6fdfaabfbf29a9f6bbf106c83c82c1d838
Author: Vivien Didelot <vivien.didelot@savoirfairelinux.com>
Date:   Tue Jun 21 15:28:26 2016 -0400

    bridge: vlan: fix a few "fdb" typos in vlan doc

    Signed-off-by: Vivien Didelot <vivien.didelot@savoirfairelinux.com>
---
 bridge/vlan.c     | 2 +-
 man/man8/bridge.8 | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/bridge/vlan.c b/bridge/vlan.c
index 717025a..a8a2e1d 100644
--- a/bridge/vlan.c
+++ b/bridge/vlan.c
@@ -299,6 +299,6 @@ int do_vlan(int argc, char **argv)
 	} else
 		return vlan_show(0, NULL);
 
-	fprintf(stderr, "Command \"%s\" is unknown, try \"bridge fdb help\".\n", *argv);
+	fprintf(stderr, "Command \"%s\" is unknown, try \"bridge vlan help\".\n", *argv);
 	exit(-1);
 }
diff --git a/man/man8/bridge.8 b/man/man8/bridge.8
index 4d5d743..3024eb2 100644
--- a/man/man8/bridge.8
+++ b/man/man8/bridge.8
@@ -551,8 +551,8 @@ device is the bridge device.
 .BI master
 the vlan is configured on the software bridge (default).
 
-.SS bridge vlan delete - delete a forwarding database entry
-This command removes an existing fdb entry.
+.SS bridge vlan delete - delete a vlan filter entry
+This command removes an existing vlan filter entry.
 
 .PP
 The arguments are the same as with
-- 
1.8.3.1

